configs/upgrader/deepin-upgrade-manager-tool.desktop

```
@@ -3,6 +3,6 @@ Name=deepin upgrade manager tool
 Comment=deepin upgrade manager tool
 Exec=/usr/bin/deepin-upgrade-manager-tool --action=notify
 NoDisplay=true
-OnlyShowIn=Deepin
+OnlyShowIn=DDE
```

如果20下有相同需求的话，建议写成
```suggestion
OnlyShowIn=DDE;Deepin
```
同时兼容两个写法。

---

done